extract_data.py

```diff
@@ -20,48 +20,13 @@ def extract_and_collect_data_from_generated_files(directory: str, has_dedicated_
 
     chassis, mobo, cpu, dimms, gpu, disks, psu = extract_data(directory, has_dedicated_gpu, gpu_in_cpu,
                                                               cleanup=cleanup, verbose=verbose, unpack=False)
-    no_dimms_str = "decode-dimms was not able to find any RAM details"
 
     # the None check MUST come before the others
```

Comment:
Not needed anymore
```suggestion
```

Reply:
I left this just because there are still chassis and mobo has a mock structure with None values in case they're not detected, so makes sense for them. This is the consideration I made earlier